.github/workflows/aws-resources-clean.yml

```diff
@@ -40,8 +40,8 @@ jobs:
       fail-fast: false
       max-parallel: 3
       matrix:
-        resource: ${{ fromJson(github.event.inputs.resource) }}
-        region: ${{ fromJson(github.event.inputs.aws_region) }}
+        resource: ${{ fromJson(github.event.inputs.resource) || '[aps, autoscaling, ec2, ecs, efs, iam, launchconfig, loadbalancer, ebs]' }}
+        region: ${{ fromJson(github.event.inputs.aws_region) || '[ us-west-2, us-east-1, us-east-2,eu-central-1,eu-west-1,eu-west-2,eu-north-1]' }}
```

Comment:
I think I agree that this configuration is unnecessary.  The cleaner should be safe to run at any time against all resources in all regions and it doesn't take that long to run.  Let's roll this back and just keep the lookback window as an option.

Reply:
IMO, the ideal use-case of cleaning up individual resources in certain AWS regions is to handle the scenario where a flaky workflow does not destroy all the resources and user would like to only delete those workflows without worrying on breaking other workflow runs across the repo.
Ex: The soak tests in `aws-otel-lambda` cluttered up a bunch of AMP workspaces which a user might want to destroy before rerunning. It would be useful to clean this by triggering the workflow with `service = amp` and `update_days_to_keep = 0` (Implying to delete all the resources that are created). This will make sure that we are not deleting other services currently being used ( C/I runs, Collector Soak tests ,etc )

Reply:
Now that you brought it up, something doesn't seem safe about 0 days to keep and I am not sure how I feel about that. It may even be a gap that we should guard against by input checking.